designs/disruption-controls-by-reason.md

```diff
@@ -0,0 +1,365 @@
+# Disruption Controls By Reason
+# Table of Contents
+1. [Disruption Controls By Reason](#disruption-controls-by-reason)
+2. [API Design](#api-design)
+   1. [Approach A: Extending the Budget API to Specify a Reason](#approach-a-extending-the-budget-api-to-specify-a-reason)
+      1. [List Approach: Multiple Reasons per Budget - Recommended](#list-approach-multiple-reasons-per-budget---recommended)
+      2. [Single Reason Approach: One Reason per Budget](#single-reason-approach-one-reason-per-budget)
+   2. [Approach B: Defining Per Reason Controls](#approach-b-defining-per-reason-controls)
+   3. [Preferred Option: List Approach](#preferred-option-list-approach)
```

Comment:
Can we make sure that this ToC matches up with the actual structure of headers in the document? Looking through the doc, I was noticing that some things that should have been "h2" were listed as "h3" and vice vresa.

Reply:
Talked about elsewhere